salt/utils/schedule.py

```diff
@@ -463,18 +464,22 @@ def run_job(self, name):
 
         if 'name' not in data:
             data['name'] = name
+        if 'run' not in data:
+            data['run'] = True
         log.info('Running Job: %s', name)
 
+        now = datetime.datetime.now()
         if not self.standalone:
             data = self._check_max_running(func,
                                            data,
                                            self.opts,
-                                           datetime.datetime.now())
+                                           now)
 
         # Grab run, assume True
         run = data.get('run', True)
```

Comment:
now duplicate?

Reply:
`now` is used in multiple places.  It's used in the call to `_check_max_running` and then it's used again when updating `_last_run` in data.  It should be the same value in both places so using a variable instead of a call to `datetime.datetime.now()`.

Reply:
The comment is about something I noticed in the area, not specific to your commit (my comments were a brain dump, intended to be discussed in the daily meeting).

The comment was about `run = data.get('run', True)` - it was no longer necessary to do that because line 470 does the same thing:

```
if 'run' not in data:
            data['run'] = True
```

The merge from master changed this line to `if data.get('run'):`, which is again unnecessary as 'run' is guaranteed to be set on line 470.